### ISSUE TITLE
clear code log button

### ISSUE THREAD

**Issue by bnlawrence** [NCAS-CMS/cf-view#30]
The cf-view window has a log of code and errors on the left. This can get long and include many things not of interest. It would be good to have a clear button (and a save button) for that window.

**Comment by sadielbartholomew** [comment]
Good idea. Noted as a useful feature request, though can't tell when I'll be able to find time to address it (maybe one for our hackathon, at the latest).
